service/src/main/kotlin/fi/espoo/evaka/shared/security/actionrule/HasUnitRole.kt

```diff
@@ -187,6 +187,28 @@ AND ad.id = ANY(:ids)
         }
     )
 
+    // For Tampere
+    fun andIsDecisionMakerForAssistanceNeedDecision() = DatabaseActionRule(
```

Comment:
This needs to be updated after rebase:

- use QueryFragment and the rule function (look at the other rules for examples)
- remove :ids parameter and check from the query

Reply:
Done!